apps/aechannel/src/aesc_force_progress_tx.erl

```diff
@@ -132,93 +122,85 @@ from_pubkey(#channel_force_progress_tx{from_id = FromId}) ->
-                                   PoI, Height, Trees) of
+    case aesc_utils:check_force_progress(Tx, Payload,
+                                   OffChainTrees, Height, Trees) of
         ok -> {ok, Trees};
         Err -> Err
     end.
 
 -spec process(tx(), aec_trees:trees(), aetx_env:env()) -> {ok, aec_trees:trees()}.
-process(#channel_force_progress_tx{addresses    = Addresses,
-                                   poi          = PoI} = Tx,
-        Trees, Env) ->
+process(#channel_force_progress_tx{offchain_trees = OffChainTrees} = Tx, Trees, Env) ->
     Height = aetx_env:height(Env),
     {value, STx} = aetx_env:signed_tx(Env),
 
     TxHash = aetx_sign:hash(STx),
-    aesc_utils:process_force_progress(Tx, Addresses,
-                                      PoI, TxHash, Height, Trees, Env).
+    aesc_utils:process_force_progress(Tx, OffChainTrees, TxHash, Height, Trees, Env).
 
 -spec signers(tx(), aec_trees:trees()) -> {ok, list(aec_keys:pubkey())}.
 signers(#channel_force_progress_tx{} = Tx, _) ->
     {ok, [from_pubkey(Tx)]}.
 
 -spec serialize(tx()) -> {vsn(), list()}.
-serialize(#channel_force_progress_tx{channel_id   = ChannelId,
-                                     from_id      = FromId,
-                                     payload      = Payload,
-                                     update       = Update,
-                                     state_hash   = StateHash,
-                                     round        = Round,
-                                     addresses    = Addresses,
-                                     poi          = PoI,
-                                     ttl          = TTL,
-                                     fee          = Fee,
-                                     nonce        = Nonce}) ->
+serialize(#channel_force_progress_tx{channel_id     = ChannelId,
+                                     from_id        = FromId,
+                                     payload        = Payload,
+                                     update         = Update,
+                                     state_hash     = StateHash,
+                                     round          = Round,
+                                     offchain_trees = OffChainTrees,
+                                     ttl            = TTL,
+                                     fee            = Fee,
+                                     nonce          = Nonce}) ->
     {version(),
      [ {channel_id    , ChannelId}
      , {from_id       , FromId}
      , {payload       , Payload}
      , {round         , Round}
      , {update        , aesc_offchain_update:serialize(Update)}
      , {state_hash    , StateHash}
-     , {addresses     , Addresses}
-     , {poi           , aec_trees:serialize_poi(PoI)}
+     , {offchain_trees, aec_trees:serialize_to_binary(OffChainTrees)}
      , {ttl           , TTL}
      , {fee           , Fee}
      , {nonce         , Nonce}
      ]}.
 
 -spec deserialize(vsn(), list()) -> tx().
```

Comment:
I expect update to:
* https://github.com/aeternity/protocol/blob/2d3246e9bed457f4399e4dde1b3e29ee3af11914/serializations.md#channel-solo-force-progress-transaction
* https://github.com/aeternity/protocol/blob/785cf1efa4e387c2a82c5ad8cb773d91fffd0106/channels/ON-CHAIN.md#channel_force_progress_tx

Reply:
Yes, protocol changes are coming